source/includes/_rest-api.md

```diff
@@ -209,4 +209,30 @@ Query string parameters
 Get translated category names for a product
 To get the translated category names, used in the product model, you can use the API endpoint:
 
-`GET https://t3-dev-api.tipser.com/v4/categories?market=se`
\ No newline at end of file
+`GET https://t3-dev-api.tipser.com/v4/categories?market=se`
+
+##Add to cart
+
+```shell
+POST https://t3-prod-api.tipser.com/v4/shoppingcart/items
+headers
+Authorization: Bearer [token]
+body
+{
+  posId: "YOUR_POS_ID",
+  productId: "583d0d22101bf54d383a1f19".
+  quantity: 1
+}
+```
+
+**Parameters**
+
+**posId:** string - required to get the commission
```

Comment:
missing description of `productId` and `quantity`. Despite the name is self explanatory, it looks weird that only posId has been mentioned.

Reply:
feel free to propose sth, this was just moved from google docks 1:1